R/compress_coverage.R

```diff
@@ -34,7 +34,7 @@ compress_coverage <- function(
   bin_path = list(
     awk = "/usr/bin/awk",
     bgzip = "/usr/bin/bgzip",
-    bgzip2 = NULL
+    bgzip2 = "bzip2"
```

Comment:
(b)gzip is not bzip2.
Use bzfile and related function to handle bzip2 compressed files